src/loaders/utils/GetMediaWikiConfigDetails.ts

```diff
@@ -0,0 +1,14 @@
+import { WindowWithMW } from 'youtube/types';
+
+declare let window: WindowWithMW;
+
+export function getMediaWikiConfigDetails() {
+	const config = window?.mw?.config;
+	const wikiId = config?.get('wgCityId');
+	const isTier3Wiki = config?.get('wgArticleFeaturedVideo')?.tier3Mapping ?? false;
+
+	return {
+		wikiId,
+		isTier3Wiki,
+	};
```

Comment:
Please specify the return type. Currently, both `wikiId` and `isTier3Wiki` are seen as `any` type:

![image](https://github.com/Wikia/jwplayer-fandom/assets/82457200/0fd267f9-e382-41f0-9a56-63489870d113)